### ISSUE TITLE
Add Discord social icon

### ISSUE THREAD

**Issue by lukemilby** [Jieiku/abridge#94]
Hey! Love the theme, I just dont know enough to figureout how to add to the social icons.

I could do it just need some guidance. Thanks!

**Comment by Jieiku** [comment]
Hello and happy to hear you love the theme! A discord icon is a good idea, it's used by many people.

I added the svg icon to the discord-icon branch I just created: https://github.com/Jieiku/abridge/tree/discord-icon

Other files that still need edits:
config.toml
templates/macros/social.html
sass/_variables.scss
sass/include/_icons.scss  (I added the svg icon here, but still need to add the @if logic)

I was able to see which places need edits by searching for a different icon, eg:

`grep --color -i --binary-files=without-match --include=\* -rnw "/home/jieiku/.dev/abridge/" -e "mastodon"`

Feel free to submit a pull request to the discord-icon branch, if you have any questions just ask, Thanks!
